src/components/Modal/Modal.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { screen } from '@testing-library/react';
+import { renderWithDeps } from '../../../jest.utils';
+import { Modal } from './Modal';
+
+describe('<Modal />', () => {
+	it('renders', () => {
+		renderWithDeps(<Modal />);
+
+		const modal = screen.getByTestId('Modal');
+
+		expect(modal).toBeInTheDocument();
+	});
+});
```

Comment:
It would be good to expand the unit tests to cover:

- Clicking on the button opens the modal
- Clicking the close button (still to be added) closes the modal

One way you might test if it's working is with `screen.getByText('some text that is in the modal').toBeVisible()`.